packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -106,6 +106,14 @@ export const useArbTokenBridge = (
 
   const [pendingWithdrawalsMap, setPendingWithdrawalMap] =
     useState<PendingWithdrawalsMap>({})
+
+  // once the l1/l2/account changes, we need to revalidate the withdrawal list in the store
```

Comment:
Should not be addressed in this PR, but do we want to "reset" most of the state when changing account? So we don't have this logic all across the app?

Reply:
Good suggestion. We do have a tracker ticket for refactoring useArbTokenBridge so I will add it there to be taken up.
https://github.com/OffchainLabs/arbitrum-token-bridge/issues/545#issuecomment-1521848810